node/pkg/adminrpc/adminserver.go

```diff
@@ -1031,7 +1031,11 @@ func (s *nodePrivilegedService) ChainGovernorResetReleaseTimer(ctx context.Conte
 		return nil, fmt.Errorf("the VAA id must be specified as \"chainId/emitterAddress/seqNum\"")
 	}
 
-	resp, err := s.governor.ResetReleaseTimer(req.VaaId)
+	if req.NumDays < 1 || req.NumDays > 7 {
```

Comment:
consider making 7 a constant and not a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)).